content/rv/administration/configuration/backups.md

```diff
@@ -5,15 +5,14 @@ weight: 70
 alwaysopen: false
 categories: ["RC Pro"]
 ---
-You can back up your Redis Cloud Pro databases to a storage bucket
-of your choosing. This article explains how to create a cloud storage
-bucket and grant permissions to use it for scheduled and on-demand
-backups. Alternatively, you may use an FTP server to store your backups
-instead.
+You can run scheduled and on-demand backups of your Redis Cloud Pro databases
+to a remote storage location. Schedule backups run every 24 hours.
 
-**Note**: Backups are only available for paid plans. If you are free
-plan user, you must upgrade your subscription to a paid plan (zero
-downtime) in order to utilize this feature.
+{{% note %}}
+Backups are only available for paid subscriptions. Customers with
```

Comment:
@bmansheim you can remove this entire paragraph as this is not the case in Pro. backup is available for trial subscriptions